ci-operator/jobs/openshift/azure-disk-csi-driver-operator/openshift-azure-disk-csi-driver-operator-master-presubmits.yaml

```diff
@@ -129,6 +129,70 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-azure-csi,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
Please manually add `optional: true` here so we don't block merges until the flakes are fixed.

Reply:
And run `make update` afterwards, it will sort the fields in the right order...

Reply:
@jsafrane  done

Reply:
@jsafrane I added `optional : true` manually to the master before running `make update`. According to the docs, this runs also `make jobs` .  However, I could not find `optional : true` in the `4.9` yaml after that, so I added them manually.

I can see below the "generated config" failing, I believe this is from the manual edit. correct me if I am wrong please 👍🏽